active/ZEP0000.md

```diff
@@ -264,130 +299,217 @@ original and new ZEPs respectively. Process ZEPs may also have a status of
-with a subject like:
-
-> `Proposal to accept ZEP <number>: <title>`
-
-**In the body of your discussion**, you should:
-
-- Link to the latest version of ZEP,
-- Briefly describe any major points of contention and how they were resolved,
-- Include a sentence like: **“If there are no substantive objections within 7 days
-  from this post, then the ZEP will be accepted**;
-
-After you create the issue, you should make sure to link the newly created
-thread in the `Discussion` section of the ZEP, so that people can find it later.
-
-Generally, the ZEP author will be the one to create this post, but anyone can
-do it – the important thing is to make sure that everyone knows when a ZEP is
-on the verge of acceptance, and give them a final chance to respond. If
-there’s some special reason to extend this final comment period beyond
-`7 days`, then that’s fine, just say so in the post. You shouldn’t do less
-than `7 days`, because sometimes people are travelling or similar and need
-some time to respond.
-
-There may be a case that a ZEP didn’t attract needed attention towards it, the
-engagement from the community is low, and `7 days` pass by. In this case,
-the author(s) of the ZEP must make necessary efforts to spread the word about
-the ZEP through [Gitter] or using the organisation [discussions] feature of
-GitHub or, if needed, creating an additional issue in the [community] or [ZEPs]
-repository. In addition, the author(s) should get in touch with the
-[Zarr Steering Council] to prevent the case that the ZEP was accepted due to
-less participation from the community.
-
-If all the above options are exhausted by the author(s), then it’s the
-responsibility of the [Zarr Steering Council] to take the final decision on the
-ZEP.
-
-In general, the goal is to make sure that the community has consensus, not
-provide a rigid policy for people to try to game. When in doubt, err on the
-side of asking for more feedback and looking for opportunities to compromise.
-
-If the final comment period passes without any substantive objections, then the
-ZEP can officially be marked `Accepted`. You should create a follow-up
-issue in [zarr-developers/zeps] repository notifying everyone
-(celebratory emoji optional but encouraged 🎉✨), and then update the ZEP by
-setting its `:Status:` to `Accepted`, and it's `:Resolution:` header to a link
-to your follow-up discussion (the last issue created) thread.
-
-If there are substantive objections, then the ZEP remains in `Draft` state,
-discussion continues as normal, and it can be proposed for acceptance again
-later once the objections are resolved.
+The Reading Phase, or Request for Comments (RFC) phase, initiates the ZEP
+proposal process, marking the introduction of the proposal to the Zarr
+community.
 
-In unusual cases, the [Zarr Steering Council] may be asked to decide whether a
-controversial ZEP is `Accepted`.
+1. **Stakeholder Engagement**:
```

Comment:
A reader might ask: Who are the stakeholders and how do I contact them?

Reply:
Mostly the ZIC and ZSC but also the broader community.
The author is supposed to use the submitted PR to spread the word to everyone, but we also have Google groups set up for notification purposes.

Reply:
:+1: to defining stakeholders clearly.